e2e/portalicious/tests/ViewFilterAndSortRegistrations/FilterRegistrationsByDate.spec.ts

```diff
@@ -47,21 +47,46 @@ test('[34947] Filter registrations by Date selection', async ({ page }) => {
   const registrations = new RegistrationsPage(page);
   const tableComponent = new TableComponent(page);
   // Act & Assert
-  await test.step('Filter registrations columns by date', async () => {
+  await test.step('Filter registration created column by "Equals" date', async () => {
     // Filter Registration column by date
     await tableComponent.filterColumnByDate({
       columnName: 'Registration created',
       day,
+      range: 'Equals',
     });
     await tableComponent.validateAllRecordsCount(4);
+    await tableComponent.clearAllFilters();
     // Filter by different date
     const diffDay = currentDate.getDate() === 1 ? 2 : 1;
     if (diffDay !== undefined) {
       await tableComponent.filterColumnByDate({
         columnName: 'Registration created',
         day: diffDay,
+        range: 'Equals',
       });
     }
     await registrations.validateRegistrationIsNotPresent();
+    await tableComponent.clearAllFilters();
+  });
+
+  await test.step('Filter registration created column by "Less than" date', async () => {
```

Comment:
Maybe it makes sense to have a step where we test a filter which does returns registartions. For example test for an equal of the current day. 

```
await tableComponent.filterColumnByDate({
        columnName: 'Registration created',
        day: currentDay,
        range: 'Equals',
      });
``